src/js/components/Select/__tests__/__snapshots__/Select-multiple-test.js.snap

```diff
@@ -489,7 +489,7 @@ exports[`Select Controlled multiple 1`] = `
 <button
   aria-expanded="false"
   aria-haspopup="listbox"
-  aria-label="Open Drop; Selected: "
+  aria-label="Open Drop; 0 selected"
```

Comment:
Can we change Select so when `multiple` is true the aria-label reads "x options selected" or "1 option selected" if there is only one?